sklearn/feature_extraction/text.py

```diff
@@ -424,8 +426,8 @@ class HashingVectorizer(BaseEstimator, VectorizerMixin, TransformerMixin):
 
       .. deprecated:: 0.20
          ``stop_words='english'`` is deprecated in version 0.20 and will be
-        removed in 0.22 as the current list is broken, leading to unexpected
-        results.
+        removed in 0.22 as the current list is controversial, leading to
```

Comment:
"and may lead to". There are also a number of cases where the mentioned issue will have no practical impact.

Reply:
Done :)